lib/unlink.js

```diff
@@ -5,21 +5,35 @@ const fs = require('fs');
 const del = require('del');
 const pify = require('pify');
 const userHome = require('user-home');
+const resolveAlfredPrefs = require('resolve-alfred-prefs');
 
 const fsP = pify(fs);
 
 const idRegexp = /<key>bundleid<\/key>[\s]*<string>(.*?)<\/string>/;
 
+const alfred3 = [
+	'Library/Application Support/Alfred 3/Workflow Data',
+	'Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Data'
+];
+
+const alfred = [
+	'Library/Application Support/Alfred/Workflow Data',
+	'Library/Caches/com.runningwithcrayons.Alfred/Workflow Data'
+];
+
 // Remove the symlink
 const unlink = dir => del(path.join(dir), {force: true});
 
 // Cleanup config and cache data
 const cleanup = dir => fsP.readFile(path.join(dir, 'info.plist'), 'utf8')
     .then(content => idRegexp.exec(content)[1])
-    .then(bundleid => Promise.all([
-		unlink(path.join(userHome, 'Library/Application Support/Alfred 3/Workflow Data', bundleid)),
-		unlink(path.join(userHome, 'Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Data', bundleid))
-    ]));
+    .then(async bundleid => {
+			const {version} = await resolveAlfredPrefs();
```

Comment:
Can you return a `Promise` here instead of using `async/await`? I don't want to drop older node versions for now.

```
return resolveAlfredPrefs()
	.then(prefs => {
		const dirs = dirMap.get(prefs.version || 'default');

		return Promise.all(dirs.map(dir => unlink(path.join(userHome, item, bundleid))));
	});
```